static/ngx_http_brotli_static_module.c

```diff
@@ -227,6 +227,14 @@ static ngx_int_t handler(ngx_http_request_t* req) {
     return NGX_DECLINED;
   }
 
+  if (cfg->enable == NGX_HTTP_BROTLI_STATIC_ON) {
```

Comment:
Not sure if this should be conditional here.
General rule - if compressed entry is going to be server - it must have vary header.